docs/step-by-step/step-03.md

```diff
@@ -13,160 +14,79 @@ and performance](step-01.md).
+gives you a ton of value at no cost:
 
-> Learn more about all the features within Nodes View and what charts/metrics are represented there in our
-> [documentation](/docs/netdata-cloud/nodes-view.md).
+> Netdata Cloud gives you real-time visibility for your entire infrastructure. With Netdata Cloud, you can run all your
+> distributed Agents in headless mode _and_ access the real-time metrics and insightful charts from their dashboards.
+> View key metrics and active alarms at-a-glance, and then seamlessly dive into any of your distributed dashboards
+> without leaving Cloud's centralized interface.
 
-You can visit Nodes View by navigating to `https://netdata.cloud/console` in your browser. Or, you can click on the
-**Nodes View** button in any Netdata dashboard. If you're not logged in to Netdata Cloud yet, you'll be asked to log in
-first.
+You can as made nodes and team members as you need, and as our free and open source Agent gets better with more
+features, new collectors for more applications, and improved UI, so will Cloud.
 
-![Animated GIF of loading the Nodes
-View](https://user-images.githubusercontent.com/1153921/65066750-d7e54180-d939-11e9-9415-a8556ed99a02.gif)
+## Get started with Netdata Cloud
 
-The Nodes View shows an aggregated list of the nodes you connected to Netdata Cloud, and shows at-a-glance health status
-for each.
+Signing in, onboarding, and claiming your first nodes only takes a few minutes, and we have a [Get started with
+Cloud](https://learn.netdata.cloud/docs/cloud/get-started) guide to help you walk through every step.
 
-Click on any of the boxes representing your nodes to see real-time, per-second charts of essential metrics in the **Node
-overview** sidebar.
+Or, if you're feeling confident, dive right in.
 
-![output-Peek 2019-09-17 11-00
-mp4](https://user-images.githubusercontent.com/1153921/65067327-192a2100-d93b-11e9-9824-80e142ac62c5.gif)
+<p><a href="https://app.netdata.cloud" className="button button--lg">Sign in to Cloud</a></p>
 
-You can also view raised alarms and see real-time metrics from a [select number of
-services/applications](/docs/netdata-cloud/nodes-view.md#services-available-in-the-nodes-view) using the various tabs
-available in the node overview sidebar.
+When you finish that guide, circle back to this step in the tutorial to learn how to use the Visited Nodes feature on
+top of Cloud's centralized web interface.
 
-If you add a large number of nodes to the Nodes View, you may want to look into the different view and sorting options.
-You can choose between **full**, **compact**, and **detailed** view modes.
+## Navigate between dashboards with Visited nodes
 
-![Animated GIF of the various view
-modes](https://user-images.githubusercontent.com/1153921/65068318-4bd51900-d93d-11e9-8720-b3bd76809d16.gif)
+To add nodes to your Visited nodes menu, you first need to navigate to that node's dashboard, then click the **Sign in**
```

Comment:
The capital Vs look jarring and out of place. Should all of these references be "Visitied Nodes" -> possibly put it in italics to make it more obvious that it is being quoted?

Reply:
I ended up rephrasing how I write about this menu, not referencing the name of the menu as much. I think it's a different way to address your valid icky feeling.